src/clucie/core.clj

```diff
@@ -13,7 +13,9 @@
   (cond
     (string? v) v
     (integer? v) (str v)
-    (keyword? v) (name v)
+    (keyword? v) (if (some? (namespace v))
+                   (-> v str (subs 1))
```

Comment:
It may be better to commonize this stringify function.

Reply:
Yes, it's a good idea